### PR TITLE
Create .env config writer

### PR DESCRIPTION
**Pull request by vyshnavsdeepak**

### CODE REVIEW

File: lib/config.dart [hunk 1]
```diff
@@ -1,24 +1,196 @@
...
+            value = acceptStdIn(envVar);
+          }
+        } else {
+          value = acceptStdIn(envVar);
+        }
+
+        value = validateAndAcceptInput(envVar, value, validators, fix: true);
+      }
+    } // validators.forEach((validator) => validator.throwIfInvalid());
+    print("ConfigItem: $envVar = $value");
```
Comment (Allen-Cherian) [inline lib/config.dart]: Please remove the unwanted comment statements.

File: lib/config.dart [hunk 2]
```diff
@@ -1,24 +1,196 @@
...
+    return value.length >= min && value.length <= max;
+  }
+}
+
+String validateAndAcceptInput(
+    String name, String value, List<Validator> validators,
+    {bool fix = false}) {
+  var errors = validators
+      .map((validator) {
+        // validator.setFieldName(name);
```
Comment (Allen-Cherian) [inline lib/config.dart]: These are not needed, please remove.

File: lib/config.dart (hunk 1 above)
Comment (vyshnavsdeepak) [reply lib/config.dart]: Removed! @Allen-Cherian . Thanks for pointing it out

File: lib/config.dart (hunk 2 above)
Comment (vyshnavsdeepak) [reply lib/config.dart]: Removed the function definition too

**Review by Allen-Cherian — APPROVED** [review]

**Pull request merged**